.editorconfig

```diff
@@ -0,0 +1,4 @@
+﻿[*.cs]
+
+# IDE0011: 添加大括号
+csharp_prefer_braces = when_multiline
```

Comment:
is this needed?

Reply:
@taooceros

Reply:
Sorry I forget to response to this message. I add that because vs is consistently asking me to change var to type name. I don't notice that it will be pushed to the repo. Sorry.

I will be afk for these two days, so I will revise that after tomorrow.

Reply:
All good, I wasn't sure of this option. I might remove it next time I continue to review this pr, if you haven't done do.

Reply:
Thank you